compiler/pragmas.nim

```diff
@@ -821,8 +823,6 @@ proc singlePragma(c: PContext, sym: PSym, n: PNode, i: var int,
         noVal(c, it)
         incl(sym.flags, {sfThread, sfGlobal})
       of wDeadCodeElimUnused: discard  # deprecated, dead code elim always on
-      of wNoForward: pragmaNoForward(c, it)
```

Comment:
This breaks code without any deprecation period!

Reply:
I mean it was meant to be an experimental feature and those can "break at any time," but you're right, I'll fix it